spec/requests/api/v1/allocations_controller_index_spec.rb

```diff
@@ -167,6 +167,50 @@
           expect(response.body).to eq('{"error":{"locations":["may not be used in combination with `from_locations` or `to_locations` filters."]}}')
         end
       end
+
+      describe 'included relationships', :skip_before do
+        let!(:allocations) { create_list :allocation, 2, :with_moves }
+
+        before do
+          get "/api/v1/allocations#{query_params}", params: params, headers: headers
+        end
+
+        context 'when not including the include query param' do
+          let(:query_params) { '' }
+
+          let(:included) do
+            from_locations = allocations.map(&:from_location)
+            to_locations = allocations.map(&:to_location)
+            moves = allocations.flat_map(&:moves)
+
+            from_locations + to_locations + moves
+          end
+
+          it 'returns the default includes' do
+            expect(response_json).to have_includes(included)
+          end
+        end
+
+        context 'when including the include query param' do
+          let(:query_params) { '?include=foo.bar,from_location' }
+
+          let(:included) { allocations.map(&:from_location) }
+
+          it 'returns the valid provided includes' do
+            expect(response_json).to have_includes(included)
+          end
+        end
+
+        context 'when including an empty include query param' do
+          let(:query_params) { '?include=' }
```

Comment:
can you also add a test for a nil value as well?

Reply:
`nil.to_s` is '' so is equivalent to https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/483/files#diff-ca0a918cadf1b72e0d225d2948b64e0eR178?

Reply:
Added a test.